### PR TITLE
pwvucontrol: init at 0.2

### LINKED ISSUE #248594: Package request: pwvucontrol
**Project description**

Volume control applet for Pipewire, written in Rust.

**Metadata**

* homepage URL: https://github.com/saivert/pwvucontrol/
* source URL:
* license: mit, bsd, gpl2+ , ...
* platforms: unix, linux, darwin, ...

### PR DESCRIPTION
**Pull request by figsoda**
https://github.com/saivert/pwvucontrol

## Description of changes

closes #248594 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by LudovicoPiero** [comment]
Result of `nixpkgs-review pr 248602` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pwvucontrol</li>
  </ul>
</details>

File: pkgs/applications/audio/pwvucontrol/default.nix [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    libadwaita
+    pango
+    pipewire
+    wireplumber
+  ];
+
+  meta = with lib; {
+    description = "Pipewire Volume Control";
+    homepage = "https://github.com/saivert/pwvucontrol";
+    license = licenses.gpl3Plus;
```
Comment (LudovicoPiero) [inline pkgs/applications/audio/pwvucontrol/default.nix:67]: Can you also add the `mainProgram` in here?

**Comment by lucasew** [comment]
Result of `nixpkgs-review pr 248602` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pwvucontrol</li>
  </ul>
</details>

**Comment by lucasew** [comment]
image

Works nice

Only thing that I am not sure it's working properly is the GTK theme.

Maybe GTK3 themes don't work as fallback for GTK4

**Review by lucasew — APPROVED** [review]

**Comment by figsoda** [comment]
It could have something to do with upstream using libadwaita, which IIRC doesn't use all the themes

**Review by PedroHLC — APPROVED** [review]
image

Looks even funkier in my qt-breeze-forced-through-debug-envvars-GTK4-theme

**Pull request merged**